Uppsala/UppEt10.xml

```diff
@@ -0,0 +1,445 @@
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a7">
+                                    <locus from="152rb" to="152vb"/>
+                                    <desc type="GuestText" ><title type="complete" ref="LIT7066NiceneCreedAmh.xml"/>
+                                    </desc>
+                                    <q xml:lang="am"><seg part="I">
+                                        እናምናለን፡ በሀንድ፡ አምላክ፡ እግዚአብሔር፡ <add place="above">አባት።</add>
+                                        ኍሉ፡ የሚቻለው፡ ሰማይና፡ ምድርን፡ የፈጠረ።
+                                        የሚታይንም፡ የማይታይንም፡ ኍሉ።
+                                        እናምናለን፡ በሀንድ፡ እግዚእ፡ ኢየሱስ፡ ክርስቶስ፡ የአባት፡ ሀንድ፡ ልጅ።
+                                    </seg></q>
+                                    <q xml:lang="gez"><seg part="F">
+                                        <locus target="#152vb"/>
+                                        እናምናለን፡ ሀንድ፡ ጥምቀት፡ ለኀጢአት፡ ይቅሬታ። ንጸናለን፡ የሙታንን፡ መነሳት። የሚመጻን፡ 
+                                        ዓለም፡ ሕይወት፡ አሜን።
+                                    </seg></q>
+                                </item>
+                                
+                                <item xml:id="a8">
+                                    <locus target="#152v"/>
+                                    <desc type="OwnershipNote">
+                                    </desc>
+                                    <q xml:lang="gez">
+                                        ዛቲ፡ መጽሐፈ፡ አርጋኖን፡ <del rend="effaced" unit="chars" quantity="7"/> ክርስቶስ፡ 
+                                        ዘአጥረያ፡ በንዋዩ፡ መድኀኒተ፡ ነፍስ፡ ትኵኖ፡ በዓለም፡ ሐዳስ፡ ማርያም፡ እሙ፡ ለኢየሱስ፡ ክርስቶስ።
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">Paper sticker on the inside of the front board.</desc>
+                                    <q xml:lang="sv">
+                                        Upsala Univ. Bibliotek<lb/>
+                                        Etiopiska<lb/>
+                                        Kat. no X.<lb/>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="ir"/>
+                                    <desc type="StampExlibris">Note in Amharic and Swedish, specifying the content of the manuscript
+                                        and signalling the ownership of <persName ref="PRS2680Bergman">August Bergman</persName>,
+                                        providing his name in Ethiopic characters (<foreign xml:lang="gez">'በርይማን'</foreign>, 
+                                        <foreign xml:lang="gez-tr">Barymān</foreign>).</desc>
```

Comment:
Is it both in Ethiopic and Turkish character or is it only in Ethiopic characters according to the Turkish pronounciation (Barymān)?